### ISSUE TITLE
[FR] Allow to listen to multiple exchanges

### ISSUE THREAD

**Issue by antonincms** [Templum/rabbitmq-connector#33]
**Is your feature request related to a problem? Please describe.**
In my workflow, we'd like to react to existing events, pushed by different microservices (let say A and B) which commit their actions in different Exchange (AEx and BEx). 

**Describe the solution you'd like**
I would nice think to have a list of echanges, instead of only one :
```yaml
RMQ_EXCHANGES : 
  - AEx
  - BEx
```
 Maybe more precisely, each exchange could have their topics to listen ?
```yaml
RMQ_EXCHANGES:
- name: AEx
  topics:
  - Foo
  - Bar
- name: BEx
  topics:
  - Dead
  - Beef
```
Which can also be writen 
```yaml
RMQ_EXCHANGES:
- name: AEx
  topics: [Foo, Bar]
- name: BEx
  topics: [Dead, Beef]
```
**Describe alternatives you've considered**
Having some kind of worker to copy-paste events from AEx and BEx to OpenFaasEx, but this is neither convenient  nor a good practice.

**Comment by Templum** [comment]
Hi @AntoninCamus ,

I like your proposal especially when the goal of the using developer is to minimize changes on existing services, also implementing it should be more or less straight forward.

So I consider to implement it, I will update this issue when I find the necessary time to work on this matter,

**Comment by Templum** [comment]
Is there anyone that would be willing to test out the new approach? I more or less finished work on this in #72. The only thing missing here is the updated documentation and an updated integration test.

Please reach out to me or post here if you are interested.

**Comment by Templum** [comment]
Was merged today, will release a Version 1.0.0 release in the next few days.

**Comment by Templum** [comment]
Was released with [V1.0.0]( https://github.com/Templum/rabbitmq-connector/tree/v1.0.0). Should be available soon in dockerhub
